test/unit/tracks/text-track-settings.test.js

```diff
@@ -9,6 +9,16 @@ const tracks = [{
   label: 'test'
 }];
 
+const defaultSettings = {
```

Comment:
I think we are in agreement.
The dropdowns shouldn't have any defaults or "empty values" and each setting should have an option of whether it will be used.